### PR TITLE
Adds system check for forced SSL connection

### LINKED ISSUE #7279: New system check to Warn users if force_ssl is not yet enabled
It seems that piwik does not automatically redirect to https.
Shouldn't that be done to increase safety? Protect the login credentials as well as all that sensitive user data...

### PR DESCRIPTION
**Pull request by sgiehl**
Maybe the message shown if force_ssl is disabled can be further improved...

fixes #7279

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -211,6 +211,8 @@
         "FileIntegrityWarningReuploadBis": "Try to reupload all the Matomo files in BINARY mode.",
         "First": "First",
         "Flatten": "Flatten",
+        "ForcedSSL": "Forced SSL Connection",
+        "ForceSSLRecommended": "We recommend using Matomo over secure SSL connections only. To disallow unsecure access over http add `force_ssl = 1` to `General` section in Matomo config.",
```
Comment (diosmosis) [inline lang/en.json]: Think `force_ssl = 1` and `General` don't need to be translated correct? And are the backticks supposed to be there or should there be `<code>` elements (inserted via placeholder)?

Comment (diosmosis) [reply lang/en.json]: Would change "disallow unsecure" to "prevent insecure", otherwise sounds good to me.

**Review by diosmosis — CHANGES_REQUESTED** [review]

File: plugins/Diagnostics/Diagnostic/ForceSSLCheck.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        $forceSSLEnabled = (Config::getInstance()->General['force_ssl'] == 1);
+
+        if ($forceSSLEnabled) {
+            return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_OK));
+        }
+
+        $comment = $this->translator->translate('General_ForceSSLRecommended');
+
+        return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_WARNING, $comment));
```
Comment (diosmosis) [inline plugins/Diagnostics/Diagnostic/ForceSSLCheck.php]: Also, should this check run if matomo isn't currently on `https`? If it isn't and a user adds `force_ssl = 1`, they may be confused why it doesn't work.

Comment (fdellwing) [reply plugins/Diagnostics/Diagnostic/ForceSSLCheck.php]: It should be checked and warned about! But with another text indicating the fact that a certificate need to be deployed first (with a link to eff's org maybe?).

File: lang/en.json (hunk 1 above)
Comment (sgiehl) [reply lang/en.json]: changed.

**Pull request merged**